supabase/migrations/020_anime_covers_storage.sql

```
@@ -0,0 +1,33 @@
+    10485760,
+    ARRAY['image/jpeg', 'image/png', 'image/webp']
+)
+ON CONFLICT (id) DO NOTHING;
+
+-- 公開読み取りポリシー
+CREATE POLICY "anime-covers are publicly accessible"
+ON storage.objects FOR SELECT
+USING (bucket_id = 'anime-covers');
+
+-- 認証済みユーザーのアップロードポリシー（サーバー側で管理者チェック）
+CREATE POLICY "authenticated users can upload anime covers"
```

SQLFluff (4.1.0)

[error] Do not use special characters in identifiers.

(RF05)